packages/core/src/interpreter.ts

```diff
@@ -589,21 +577,12 @@ export class Interpreter<
       isSCXMLErrorEvent(_event) &&
       !this.state.nextEvents.some((nextEvent) => nextEvent === _event.name)
     ) {
+      // Error event unhandled by machine
       this.handleErrorEvent(_event);
+      this.stop();
```

Comment:
@davidkpiano this is a somewhat surprising change in this PR, was there any specific reason for it to be included here? 🤔